helm/crossplane/templates/crd-install/crd-np.yaml

```diff
@@ -1,4 +1,24 @@
 {{- if .Values.giantswarm.crds.install }}
+{{- if .Capabilities.APIVersions.Has "cilium.io/v2/CiliumNetworkPolicy" }}
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: {{ include "crdInstall" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    # create hook dependencies in the right order
+    "helm.sh/hook-weight": "-7"
+    {{- include "crdInstallAnnotations" . | nindent 4 }}
+  labels:
+    app.kubernetes.io/component: {{ include "crdInstall" . | quote }}
+    {{- include "labels.selector" . | nindent 4 }}
+    role: {{ include "crdInstallSelector" . | quote }}
+spec:
```

Comment:
Spec of this policy looks the same as of the previous one `allow-apiserver-access`. Can't we rely on the previous one only?

Reply:
Forgot to remove. They are the same currently, so we can use the one generated in pre-hook.